featureflags/client.py

```diff
@@ -177,7 +177,7 @@ def authenticate(self):
             verify = self._config.tls_trusted_cas_file
 
         client = Client(base_url=self._config.base_url, verify_ssl=verify,
-                        raise_on_unexpected_status=True)
+                        raise_on_unexpected_status=True, httpx_args=self._config.httpx_args)
```

Comment:
Shorten line length to get CI passing for `flake8` 
```suggestion
                        raise_on_unexpected_status=True,
                        httpx_args=self._config.httpx_args)
```

Reply:
I've gone ahead and merged this into a feature branch and I will make the change there.